### ISSUE TITLE
Bug in 7.0.0 in gjs files

### ISSUE THREAD

**Issue by karl-sjogren** [ember-template-lint/ember-template-lint#3134]
After upgrading to 7.0.0 in our project I'm getting the following error when linting `.gjs` files.

> Incorrect indentation for `<div>` beginning at L2:C4. Expected `<div>` to be at an indentation of NaN, but was found at 4. (fixable)ember-template-lint[block-indentation](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/block-indentation.md)

The indentation is correct in the file. If I break the indentation in a `.hbs` file it reports as expected.

> Incorrect indentation for `section` beginning at L179:C4. Expected `</section>` ending at L183:C16 to be at an indentation of 4 but was found at 6. (fixable)ember-template-lint[block-indentation](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/block-indentation.md)

This is my `.template-lintrc.js` file.

```js
'use strict';

module.exports = {
  extends: 'recommended',
  rules: {
    'no-bare-strings': false,
    'quotes': 'double',
    'require-button-type': false, // 2021-07-30 KJ Maybe add this back later
    'require-valid-alt-text': false,
    'require-input-label': false,
    'block-indentation': true,
    'require-media-caption': false,
    'require-each-key': true,
    'no-builtin-form-components': false,
    'no-at-ember-render-modifiers': false
  }
};
```

These are my installed packages.

```js
{
  "name": "am",
  "version": "0.0.0",
  "packageManager": "yarn@4.6.0",
  "private": true,
  "devDependencies": {
    "@babel/core": "^7.26.9",
    "@babel/eslint-parser": "^7.26.8",
    "@babel/plugin-proposal-decorators": "^7.25.9",
    "@babel/preset-env": "^7.26.9",
    "@ember/optional-features": "^2.2.0",
    "@ember/render-modifiers": "^3.0.0",
    "@ember/string": "^4.0.1",
    "@ember/test-helpers": "^5.1.0",
    "@eslint/js": "^9.21.0",
    "@glimmer/component": "~1.1.2",
    "@glimmer/tracking": "~1.1.2",
    "@microsoft/signalr": "^8.0.7",
    "@stylistic/eslint-plugin-js": "^4.2.0",
    "@stylistic/stylelint-plugin": "^3.1.2",
    "@zxing/library": "^0.21.3",
    "babel-plugin-module-extension": "^0.1.3",
    "broccoli-asset-rev": "^3.0.0",
    "broccoli-funnel": "^3.0.8",
    "broccoli-merge-trees": "^4.2.0",
    "chart.js": "4.4.8",
    "chartjs-plugin-zoom": "2.2.0",
    "clndr2": "2.1.0",
    "date-fns": "^4.1.0",
    "dragula": "^3.7.3",
    "ember-auto-import": "^2.10.0",
    "ember-cli": "6.2.2",
    "ember-cli-app-version": "^7.0.0",
    "ember-cli-autoprefixer": "^2.0.0",
    "ember-cli-babel": "^8.2.0",
    "ember-cli-bundle-analyzer": "^1.0.0",
    "ember-cli-dependency-checker": "^3.3.3",
    "ember-cli-htmlbars": "^6.3.0",
    "ember-cli-inject-live-reload": "^2.1.0",
    "ember-cli-sass": "^11.0.1",
    "ember-cli-terser": "^4.0.2",
    "ember-fetch": "^8.1.2",
    "ember-in-viewport": "^4.1.0",
    "ember-intl": "^7.1.4",
    "ember-leaflet": "^5.1.3",
    "ember-load-initializers": "^3.0.1",
    "ember-page-title": "^9.0.1",
    "ember-ref-bucket": "^5.0.7",
    "ember-resolver": "^13.1.0",
    "ember-source": "6.2.0",
    "ember-template-imports": "^4.3.0",
    "ember-template-lint": "^7.0.0",
    "ember-user-activity": "^8.0.1",
    "eslint": "^9.20.0",
    "eslint-plugin-ember": "^12.5.0",
    "eslint-plugin-file-progress": "^3.0.1",
    "eslint-plugin-jest": "^28.11.0",
    "eslint-plugin-jsdoc": "^50.6.3",
    "eventemitter3": "^5.0.1",
    "globals": "^16.0.0",
    "hammerjs": "^2.0.8",
    "is-in-ci": "^1.0.0",
    "jest": "^29.7.0",
    "jest-environment-jsdom": "^29.7.0",
    "jquery": "^3.7.1",
    "js-levenshtein": "^1.1.6",
    "leaflet": "^1.9.3",
    "loader.js": "^4.7.0",
    "mermaid": "^11.4.1",
    "nouislider": "^15.8.1",
    "numeral": "^2.0.6",
    "onscan.js": "^1.5.2",
    "panzoom": "^9.4.3",
    "patternomaly": "^1.3.2",
    "postcss": "8",
    "postcss-scss": "^4.0.9",
    "qs": "^6.14.0",
    "sass": "^1.85.1",
    "stylelint": "^16.15.0",
    "stylelint-config-standard": "^37.0.0",
    "stylelint-config-standard-scss": "^14.0.0",
    "swedish-phone-number-formatter": "^1.0.1",
    "testem-multi-reporter": "1.2.0",
    "tracked-built-ins": "^4.0.0",
    "webpack": "^5.98.0",
    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz",
    "xmlbuilder": "^15.1.1"
  },
  "resolutions": {
    "@xmldom/xmldom": "^0.9.7",
    "mout": "^1.2.4",
    "terser": "^5.37.0",
    "glob-parent": "^6.0.2",
    "ansi-regex": "^5.0.1",
    "ansi-html": "^0.0.8",
    "minimist": "^1.2.8",
    "async": "^2.6.4",
    "minimatch": "^7.4.6",
    "json5": "^2.2.3",
    "cookie": "^1.0.2"
  },
  "engines": {
    "node": ">= 20"
  },
  "ember": {
    "edition": "octane"
  }
}
```

As a temporary workaround I've disabled `block-indentation` for `.gjs` files.
